### PR TITLE
Parse Verdef information

### PR DESCRIPTION
**Pull request by akawashiro**

### CODE REVIEW

File: symtab_builder.cc [hunk 1]
```diff
@@ -49,7 +49,7 @@ bool SymtabBuilder::Resolve(const std::string& name, const std::string& soname,
         Syminfo* found = NULL;
         for (int i = 0; i < src_syms_.size(); i++) {
             if (src_syms_[i].name == name && src_syms_[i].soname == soname && src_syms_[i].version == version) {
-                found = &src_syms_[i];
+                if (found == NULL || IsDefined(*src_syms_[i].sym)) found = &src_syms_[i];
```
Comment (shinh) [inline symtab_builder.cc:52]: Hmm I'm not sure if this is right. When there are multiple defined symbols, shouldn't the first symbol win?

I think I'd suggest we should change back `src_syms_` from vector to map (keyed by a tuple of name, soname, and version)

Comment (akawashiro) [reply symtab_builder.cc]: > shouldn't the first symbol win?
I think so.
> change back src_syms_ from vector to map
That sounds good.

File: symtab_builder.h [hunk 2]
```diff
@@ -41,7 +41,8 @@ class SymtabBuilder {
         uintptr_t index;
     };
 
-    std::vector<Syminfo> src_syms_;
+    // map from (name, soname, version) to Versym and Sym
```
Comment (shinh) [inline symtab_builder.h:44]: Thanks for this comment!

**Review by shinh — APPROVED** [review]
LGTM!!

**Pull request merged**